src/Http/Middleware/ForbidBannedUser.php

```diff
@@ -45,11 +45,20 @@ public function __construct(Guard $auth)
     public function handle($request, Closure $next)
     {
         $user = $this->auth->user();
+        $redirect_url = config('ban.redirect_url', false);
```

Comment:
I'd prefer to have `null` default value and check for `$redirectUrl === null` to determine what case should be used. First of all by this way we will avoid mixed variable types `string|bool` and will use nullable string.

Reply:
Now checking for null